pages/maker/maker-layout.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Flex, Box, Type } from 'blockstack-ui'
+
+import { Page } from '@components/page'
+
+export const MakerContainer = ({children}) => (
+  <Flex
+    // maxWidth={544}
+    // px={[1, 3]}
+    mb={5}
+    mx={[0, 20]}
+    flexDirection={['column', 'column', 'column', 'row']}
+    justifyContent="space-between"
+    flexWrap="wrap"
+  >
+    <Box flexGrow={1}>
+      {children}
+    </Box>
+  </Flex>
+)
+
+//
+// Copied styles, see if exists anywhere else
+export const StyledBox = styled(Box)`
```

Comment:
This component is a little confusing, mainly the name of it. I.e. is it one of the cards on the maker page? Additionally, this also follows the comment I made about using our spacing system instead of hard-coded spacing like `32px`.

Side note / tip / nit: if you want to use our spacing system but also need to override some CSS (like you do here), you can set `defaultProps` on the component. So for this, `StyledBox.defaultProps = { p: 3 }` should work.